### DISCUSSION TITLE
[Idea] Hability to copy custom datas during object duplication

### DISCUSSION THREAD

**Discussion by PululuK** [PrestaShop/PrestaShop#36790] [Ideas]
### The problem

Some modules add custom fields in some objects, when duplicating these objects it is currently difficult to copy this data.

### The solution

Add new hooks `actionObjectDuplicateAfter` && `actionObject<ObjectClassName>DuplicateAfter`

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Makes sense! Thanks for the PR.
